_posts/2021-03-05-cloud-gov_tech_talk_-_internal_routes.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Cloud.gov Tech Talk - Internal Routes
+date: March 5, 2021
+---
+Join us on March 17 for a [tech talk to learn more about internal routes on cloud.gov](https://www.eventbrite.com/e/tech-talk-internal-routes-tickets-144475338627). Our engineers will show you how to use internal routes for secure, direct, container-to-container communication between applications.
+
+cloud.gov provides flexibility for controlling how traffic moves between applications. In this tech talk, you will learn how to use internal routes for secure, direct, container-to-container communication between applications running on cloud.gov. We will highlight common use cases for internal routes (including links to samples) as well as discuss the considerations you need to account for. We will demonstrate how you can improve the security posture of your applications, without changing application code, using internal routes with the nginx buildpack.
```

Comment:
Might make sense to remove `for secure, direct, container-to-container communication between applications running` in the second paragraph, as it might seem repetetive.